ui/src/literature/components/NumberOfResultsWithSelectedItemsNumber.jsx

```diff
@@ -14,18 +14,18 @@ function getFormattedNumberOfSelectedOrNull(numberOfSelected) {
   );
 }
 
-function NumberOfResults({ namespace, numberOfSelected }) {
+function NumberOfResults({ namespace, numberOfSelected}) {
   return (
     <span>
-      {getFormattedNumberOfSelectedOrNull(numberOfSelected)}
+      {!!numberOfSelected && getFormattedNumberOfSelectedOrNull(numberOfSelected)}
```

Comment:
Why do you need this? anyway if it's undefined or empty it will return `null`.

Reply:
If the value of _numberOfSelected_ is undefined, _getFormattedNumberOfSelectedOrNull_ does not return null, it only returns null if _numberOfSelected_ is 0. I can either change it to return 0 if the value is undefined, like this: 

```
function getFormattedNumberOfSelectedOrNull(numberOfSelected) {
  if (numberOfSelected === 0 || !numberOfSelected) {
    return null;
  }
```

or leave the above check for truthiness of the value. What do you think :)?